src/main/java/com/exactpro/th2/mstore/ProtoUtil.java

```diff
@@ -26,6 +26,8 @@
 
 import static com.exactpro.th2.common.util.StorageUtils.toCradleDirection;
 import static com.exactpro.th2.common.util.StorageUtils.toInstant;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.apache.commons.lang3.StringUtils.trimToEmpty;
 
 public class ProtoUtil {
     public static MessageToStore toCradleMessage(Message protoMessage) {
```

Comment:
Do we also need to add metadata for `Message protoMessage`?

Reply:
Message processing is redundant API and should be removed in the next major version

Reply:
OK, I see.